herd/libdir/aarch64util.cat

```diff
@@ -44,6 +44,10 @@
  * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *)
 
+let iico_data = iico_data+
```

Comment:
Might be interesting to also specify here that `iico_ctrl` and `iico_order` are transitive? This way the changes to `pick-data` would be more robust.